service/library/clean.go

```diff
@@ -35,19 +39,33 @@ func Clean(c Config) {
 	}
 
 	for _, network := range c.Networks {
-		if s, _ := model.DockerNetworkStatus(&network); s {
-			e := model.DockerNetworkRemove(&network)
+		if s, _ := model.DockerNetworkStatus(network.Name); s {
+			e := model.DockerNetworkRemove(network.Name)
 			if e != nil {
 				fmt.Println(e)
 			}
-			if s, _ := model.DockerNetworkStatus(&network); !s {
+			if s, _ := model.DockerNetworkStatus(network.Name); !s {
 				fmt.Printf("Successfully removed network %v\n", network.Name)
 			} else {
 				fmt.Printf("Network %v was not removed\n", network.Name)
 			}
 		}
 	}
 
+	for n := range NetworksToClean {
+		if s, _ := model.DockerNetworkStatus(NetworksToClean[n]); s {
+			e := model.DockerNetworkRemove(NetworksToClean[n])
+			if e != nil {
+				fmt.Println(e)
+			}
+			if s, _ := model.DockerNetworkStatus(NetworksToClean[n]); !s {
+				fmt.Printf("Successfully removed network %v\n", NetworksToClean[n])
+			} else {
+				fmt.Printf("Network %v was not removed\n", NetworksToClean[n])
+			}
+		}
+	}
+
```

Comment:
More opportunity to centralise the process into one loop using `NetworksToClean`